docs/competition/teams/process-docs/overnight-battery-loan.md

```diff
@@ -0,0 +1,13 @@
+# Overnight Battery Loan
+
+Teams often request a battery and charger for use overnight to test their robots.
+
+## Obtaining batteries
+
+Batteries and chargers should be collected from helpdesk by teams. Teams should be given a charger, and no more than 2 batteries.
```

Comment:
Also a battery bag.

I would encourage that we give teams a _fixed_ quantity of things, which will make recovering the items from them in the morning easier. Specifically, if we give everyone two batteries then we don't need to worry about whether they only had one battery or two when we collect them back - we just demand two batteries from them.